### PR TITLE
Added `avn user` documentation

### PR DESCRIPTION
**Pull request by ftisiot**
Added `avn user` documentation with different file for `avn user access-token` which has several options.
Looking for feedback around structure since this could be used for `avn service` as well

### CODE REVIEW

**Comment by hab-anka** [comment]
One general comment, I think all the commands should be on the same navigation level. Here we're nesting the `access token` under `user`, and I understand this may make sense, but from the nav perspective this is hidden and only visible via the link. 
I suggest we make them sibling in the nav, and then link from one to another - like you did in your `user `doc.

File: docs/tools/cli/user.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+Logs out the user.
+
+
+**Example:** Log out the user.      
+::
+
+  avn user logout
+
+``avn user tokens-expire``
```
Comment (hab-anka) [inline docs/tools/cli/user.rst:119]: I wonder if this command does not logically belong to the next file (`avn user access token`)?

**Review by hab-anka — COMMENTED** [review]
One general comment about structure and a few edits in comments for grammar.

**Comment by lornajane** [comment]
Fixes #135

File: docs/tools/cli/user.rst (hunk 1 above)
Comment (lornajane) [reply docs/tools/cli/user.rst:119]: It does but I don't think we can fix that with the documentation. How about signposting the other page of commands (I think there are probably a few places we'd want to do a "see also" type of command linking.

Comment (ftisiot) [reply docs/tools/cli/user.rst:119]: fixed

File: docs/tools/cli/user.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Here you'll find the full list of commands for ``avn user``.
+
+
+Create and manage users and sessions
+------------------------------------
+
+Commands for managing project users and ``avn`` client sessions.
+
+``avn user access-token``
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/user.rst:12]: Fixed

File: docs/tools/cli/user/user-access-token.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    ====================  ============  ================================  ===============  ================  ====================  ===========  ===================
+    2021-09-15T15:29:14Z  XCJ3+bgWywIh  Test token                        2592000          true              2021-08-16T15:29:14Z  192.168.1.1  aiven-client/2.12.0
+    2021-08-16T16:26:10Z  6JsKDclT3OMQ  To be used with Python Notebooks  3600             true              null                  null         null
+
+
+
+``avn user access-token revoke``
+''''''''''''''''''''''''''''''''
+
+Revokes the specified user access token.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/user/user-access-token.rst:89]: fixed thanks

**Review by lornajane — APPROVED** [review]
Thanks very much!

**Pull request merged**